### ISSUE TITLE
Latest in Maven central repo is really old

### ISSUE THREAD

**Issue by alexroussos** [salesforce-marketingcloud/FuelSDK-Java#55]
The latest build artifact available through maven is more than a year old and is missing useful changes (eg this [commit that adds Attributes](https://github.com/salesforce-marketingcloud/FuelSDK-Java/commit/96ae1916f69ebeb87127c436c39930ffe3e64293) to ETSubscriber). 

A new release would be much appreciated

**Comment by ghkarmseveer** [comment]
Agreed, but have to ask, is this code even still supported? Master branch clearly has bugs (e.g. accessType=online breaks ETClient.requestToken, see #5 ) and no new commits for 10 months

**Comment by smunuswami** [comment]
New version is released with latest changes in maven.
